app/views/unattended/report_templates/host_-_vulnerabilities.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Host - Vulnerabilities
+snippet: false
+model: ReportTemplate
+template_inputs:
+- name: Without Errata Only
+  required: true
+  input_type: user
+  description: Show only CVEs without errata for host
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: "yes"
+require:
+- plugin: foreman_openscap
+-%>
+
+<%- report_headers 'Host Name', 'Ref Id', 'Ref URL' -%>
+
+<%- load_hosts(includes: [:cves]).each_record do |host| -%>
```

Comment:
one of the stories asked for hosts filter, could you please also add it here? just define a new input of search type and use it in `load_hosts` macro as `search: input('Hosts filter')`